ballsdex/settings.py

```diff
@@ -183,10 +185,10 @@ def read_settings(path: "Path"):
     )
 
     if admin := content.get("admin-panel"):
-        settings.webhook_url = admin.get("webhook-url")
-        settings.client_id = admin.get("client-id")
-        settings.client_secret = admin.get("client-secret")
-        settings.admin_url = admin.get("url")
+        settings.webhook_url = admin.get("webhook-url") or ""
+        settings.client_id = admin.get("client-id") or ""
+        settings.client_secret = admin.get("client-secret") or ""
+        settings.admin_url = admin.get("url") or ""
```

Comment:
```suggestion
        settings.webhook_url = admin.get("webhook-url", "")
        settings.client_id = admin.get("client-id", "")
        settings.client_secret = admin.get("client-secret", "")
        settings.admin_url = admin.get("url", "")
```

Going to leave the full review to laggron, just for this - get has a default value as an arg instead of having to use a conditional